mmv1/third_party/terraform/acctest/go/resource_test_utils.go

```diff
@@ -0,0 +1,52 @@
+package acctest
```

Comment:
This file is needed. `go` folder is for the converted files from .go.erb files. Sorry for the confusion.

Reply:
This file can be removed.